grafana-dashboard/Altinity_ClickHouse_Operator_dashboard.json

```diff
@@ -3372,12 +3382,17 @@
       "seriesOverrides": [],
       "spaceLength": 10,
       "stack": false,
-      "steppedLine": false,
+      "steppedLine": true,
       "targets": [
         {
-          "expr": "chi_clickhouse_metric_PartMutation{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}",
-          "legendFormat": "{{hostname}}",
+          "expr": "chi_clickhouse_table_mutations{exported_namespace=~\"$exported_namespace\",chi=~\"$chi\",hostname=~\"$hostname\"}",
+          "legendFormat": "mutations {{database}} {{table}} {{hostname}}",
```

Comment:
Is it going to display different series for different databases/tables?

Reply:
yes 
do you want reduce it to hostname only?

Reply:
Yes, let's make it hostname level. Users may go to mutations table for the detail

Reply:
fixed in https://github.com/Altinity/clickhouse-operator/pull/659/commits/8added93490bff83fce7d9a37ff0fef64189d36e